core/src/modules/ops/export.ts

```diff
@@ -103,6 +103,7 @@ export namespace ExportOps {
         { startedAt: new Date() },
         {
           where: { id: { [Op.in]: _exports.map((e) => e.id) } },
+          returning: true,
```

Comment:
New in this version - if you want an `update` to return the effected rows, it's explicit now